src/kagglehub/handle.py

```diff
@@ -6,6 +6,9 @@
 
 from kagglehub.config import get_kaggle_api_endpoint
 
+NUM_VERSIONED_DATASET_PARTS = 4  # e.g.: <owner>/<dataset>/versions/<version>
```

Comment:
Do we want to add `versions/` prefix or not?

Pros of keeping it:
- Matches the dataset URL.

Cons of keeping it:
- Handle is different from models.

@jplotts any thoughts?